tests/test_remote.py

```
@@ -76,21 +82,23 @@ async def test_remote_send_codes_mapped(mock_ynca, mock_zone_zone3):
-async def test_remote_send_codes_raw_formats(mock_ynca, mock_zone_zone3):
+async def test_remote_send_codes_raw_formats(
+    mock_ynca: Mock, mock_zone_zone3: Mock
+) -> None:
     entity = YamahaYncaZoneRemote("ReceiverUniqueId", mock_ynca, mock_zone_zone3, {})
 
     # Setting value
     entity.send_command(["12ABCD"])
     mock_ynca.sys.remotecode.assert_called_with("12EDABCD")
 
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError):  # noqa: PT011
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `PT011`)

Remove unused `noqa` directive

(RUF100)